### DISCUSSION TITLE
Event "signIn" won't be triggered. Am I missing something?

### DISCUSSION THREAD

**Discussion by simonwidjaja** [nextauthjs/next-auth#663] [Help]
**Your question**
A have just installed the blank template with next js and next-auth. Everything is working fine except the signIn event. It will never be triggered. The createUser event works just fine. Is this a bug or am I doing something wrong?

[...nextauth].js
```
(...)
  events: {
    signIn: async (message) => { console.error('CUSTOM EVENT signIn'); },
    signOut: async (message) => { /* on signout */ },
    createUser: async (message) => { console.error('CUSTOM EVENT createUser');  },
    linkAccount: async (message) => { /* account linked to a user */ },
    session: async (message) => { /* session is active */ },
    error: async (message) => { /* error in authentication flow */ }
  }  
(...)
```

Any help appreciated

**Comment by adiyosefi** [comment]
it is occurring on the BE so u can see the logs in the terminal, not in your browser
